resource-directory/test/service.go

```diff
@@ -18,11 +18,9 @@ func MakeConfig(t *testing.T) service.Config {
 	cfg.APIs.GRPC.Config = config.MakeGrpcServerConfig(config.RESOURCE_DIRECTORY_HOST)
 	cfg.APIs.GRPC.OwnerCacheExpiration = time.Minute
 
-	cfg.Clients.AuthServer.CacheExpiration = time.Millisecond * 50
-	cfg.Clients.AuthServer.PullFrequency = time.Millisecond * 200
-	cfg.Clients.AuthServer.OwnerClaim = config.OWNER_CLAIM
-	cfg.Clients.AuthServer.Connection = config.MakeGrpcClientConfig(config.AUTH_HOST)
-	cfg.Clients.AuthServer.OAuth = config.MakeOAuthConfig()
+	cfg.Clients.IdentityServer.CacheExpiration = time.Millisecond * 50
+	cfg.Clients.IdentityServer.PullFrequency = time.Millisecond * 200
```

Comment:
I think these 2 variables could be removed too. It was used with `userDevicesCache` which was replaced by `ownerCache`

Reply:
Indeed they are no longer needed, so they have been removed.